core/src/main/java/pl/agh/edu/actor/utils/Fonts.java

```diff
@@ -0,0 +1,33 @@
+package pl.agh.edu.actor.utils;
+
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import pl.agh.edu.actor.HotelSkin;
+
+public class Fonts {
```

Comment:
same as colors but i do that class so you can look up and decide what we do next